python/tests/conftest.py

```diff
@@ -15,6 +15,7 @@
 
 # Third Party
 import pytest
+from torch.utils.data import DataLoader  # Prevent DataLoader hangs
```

Comment:
why does it hang if you don't import it here?

Reply:
I have no good theory, need to talk with the team.
But TLDR after days of trying is that `torch.DataLoader` needs to be imported before` pyspark.sql`

Reply:
Added a `rikai/tests/torch/conftest.py`